client/src/components/application/DotsStepper/index.js

```diff
@@ -30,7 +30,7 @@ const DotStepper = ({ steps }) => {
   };
 
   const handleApply = () => {
-    window.location.href = '/';
+    window.location.href = '/signup';
```

Comment:
There's a way to use react-router to navigate to signup instead of setting the window location, which would be better because then the user wouldn't have to load the whole page again.
If you wrap this component in react-router's `withRouter` HOC, then you automatically have a `history` prop injected into your component, and you can use that to navigate in handleApply like this:
`history.push('/signup');`, if you destructured history from the props on line 12.
To wrap the component, usually it's done when exporting the component at the bottom of the file, like this:
`export default withRouter(dotStepper);`
You can read more about withRouter here:
https://github.com/ReactTraining/react-router/blob/master/packages/react-router/docs/api/withRouter.md

Reply:
This is the best review i ever had, so its important advice and its the first change iwill do

Reply:
Cool!  I'll look again after you make those changes.